config/initializers/dfe_analytics.rb

```diff
@@ -53,4 +53,9 @@
   # to all events we send to BigQuery.
   #
   # config.environment = ENV.fetch('RAILS_ENV', 'development')
+
+  # Schedule a maintenance window during which no events are streamed to BigQuery
+  # in the format of '22-01-2024 19:30..22-01-2024 20:30' (UTC).
+  config.bigquery_maintenance_window = "28-05-2024 20:00..28-05-2024 20:10"
```

Comment:
So for ten long minutes nothing is set to biq query?

Reply:
That's right :-) It will get send after the 10 mins is over.